src/Components/CurrentlyCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Card, Item } from 'semantic-ui-react';
+import CurrentItem from './CurrentItem';
+
+const CurrentlyCard = () => {
```

Comment:
I'm not a big fan of super long names, but I think `CurrentlyCard` is not that much self explanatory, unless you know the context itself. We should avoid using React-ish names for more propietary variable names so we can distinguish better what is ours and what is more framework native.

Reply:
I agree the naming of this component can change, right now it is just based on the base component used from semantic and the use for it as it is in the "currently playing" section.

Reply:
Yes, I assumed it was for that. That's why i was thinking something like CurrentlyPlayindCard? Or something more creative haha